com.github.robertsanseries.ciano.json

```diff
@@ -54,24 +56,75 @@
             "--without-xml",
             "--without-zlib"
           ],
-          "sources": [
-            {
-              "type": "archive",
-              "url": "https://github.com/ImageMagick/ImageMagick/archive/7.0.11-6.tar.gz",
-              "sha256": "8adc1605784653b078572b825e8cd1d3d54f8a1b4ba86b32ca253c038f7e4c37"
-            }
-          ]
+          "sources": [{
+              "type": "git",
+              "url": "https://github.com/ImageMagick/ImageMagick.git",
+              "tag": "7.1.1-38",
+              "commit": "b0ab92265bab638e6ecd2f18b45977c38771c671"
+          }]
         },
         {
             "name": "chrpath",
             "cleanup": ["*"],
-            "sources": [
-                {
-                  "type": "archive",
-                  "url": "http://deb.debian.org/debian/pool/main/c/chrpath/chrpath_0.16.orig.tar.gz",
-                  "sha256": "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
-                }
-            ]
+            "sources": [{
+                "type": "archive",
+                "url": "http://deb.debian.org/debian/pool/main/c/chrpath/chrpath_0.16.orig.tar.gz",
+                "sha256": "bb0d4c54bac2990e1bdf8132f2c9477ae752859d523e141e72b3b11a12c26e7b"
+            }]
+        },
+        {
+            "name": "x264",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--enable-static",
+                "--enable-pic"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "http://deb.debian.org/debian/pool/main/x/x264/x264_0.164.3108+git31e19f9.orig.tar.gz",
+                "sha256": "41606cb8e788a7f8c4514290646d4ba5c7bc68d9e1ccd1a73f446a90546913eb"
+            }]
+        },
+        {
+            "name": "aac",
+            "buildsystem": "simple",
+            "build-commands": [
+                "autoreconf -fiv",
+                "./configure --prefix=/app --disable-shared",
+                "make",
+                "make install"
+            ],
```

Comment:
This should use autotools instead of simple, you can add autoreconf -fiv as a shell source https://docs.flatpak.org/en/latest/module-sources.html#shell-and-script-sources